dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2019 IBM Corporation and others.
+ * Copyright (c) 2016, 2020 IBM Corporation and others.
```

Comment:
this file is now same as before you started making changes... copyright should go back to original date (2016, 2018)